resolver-dns/src/main/java/io/netty/resolver/dns/DnsServerAddresses.java

```diff
@@ -70,8 +71,8 @@
         } else {
             Collections.addAll(
                     defaultNameServers,
-                    new InetSocketAddress("8.8.8.8", DNS_PORT),
-                    new InetSocketAddress("8.8.4.4", DNS_PORT));
+                    SocketUtils.socketAddress("8.8.8.8", DNS_PORT),
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Make this IP "8.8.8.8" address configurable. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS1313)